src/Context/Notes/NoteProvider.jsx

```diff
@@ -28,6 +32,7 @@ const NoteProvider = ({ children }) => {
     deletedNotes: JSON.parse(localStorage.getItem("deletedNotes")) ?? [],
     allNotes: JSON.parse(localStorage.getItem("savedNotes")) ?? [...demoNotes],
     pinnedNote: JSON.parse(localStorage.getItem("pinnedNotes")) ?? [],
+    archivedNotes: JSON.parse(localStorage.getItem("archivedNotes")) ?? [],
   });
```

Comment:
Suggestion: you could create a common obj to store on localStorage and extract different list of note from that one object.

Reply:
Oh yes, I didn't think about this. Thank you for pointing it out.